apps/src/netsim/NetSimBitLogPanel.html.ejs

```diff
@@ -21,31 +21,29 @@
     <%
   }
 %>
-<div>
-  <div class="scroll-area">
-    <div class="packet">
-      <table>
-        <tbody>
-          <%
-            logRow(EncodingType.ASCII, dataConverters.binaryToAscii(binary, chunkSize));
+<div class="scroll-area bit-log-scroll-area">
+  <div class="packet">
+    <table>
+      <tbody>
+        <%
+          logRow(EncodingType.ASCII, dataConverters.binaryToAscii(binary, chunkSize));
```

Comment:
This diff is a mess to read.  It removes the outer `<div>` and moves the "Read Wire" button (at the bottom of the file) into the scroll area - which is unlikely to scroll except on very, very cramped screens.